preferences/models.py

```diff
@@ -13,7 +11,7 @@ class SabbaticalLength(models.TextChoices):
         NONE = 'NONE', _('None')
 
     professor = models.OneToOneField(
-        AppUser,
+        'users.AppUser',
```

Comment:
Is there a reason for quotation marks around the class type? I am just unaware of  the usage here

Reply:
There is a circular import dependency between AppUser and Preferences now which causes a django error. So by changing it to use quotation marks, the professor can still reference the appUser but the import dependency is gone